eng/pipelines/templates/stages/archetype-cpp-release.yml

```diff
@@ -86,53 +86,69 @@ stages:
                     steps:
                       - checkout: self
 
+                      - pwsh: |
+                          $branchName = "azure-sdk-for-cpp-${{ artifact.Name }}-$(Build.BuildId)"
+                          if ('$(VcpkgForkBranchName)') { 
```

Comment:
@vhvb1989 - based on discussion with Daniel, here's the convention we want to use for the branch name, when triggering a release pipeline:

`azure-sdk-for-cpp-<milestone>`

So, for the next one:
`azure-sdk-for-cpp-Jan-2021`

Also, make sure to release the packages sequentially.

Reply:
FYI @RickWinter

Reply:
Also fyi @katmsft, @Jinming-Hu, @vinjiang